test/services/AddressLookupServiceSpec.scala

```diff
@@ -33,43 +33,72 @@ import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
 
 class AddressLookupServiceSpec extends SpecBase with FutureAwaits with DefaultAwaitTimeout  {
-  val fixedUUID: String = "12"
+
   val mockSdilConnector = mock[AddressLookupConnector]
   implicit val hc = HeaderCarrier()
-  val service = new AddressLookupService(mockSdilConnector, frontendAppConfig){
-    override def generateId: String = fixedUUID
-  }
+  val service = new AddressLookupService(mockSdilConnector, frontendAppConfig)
 
   "getAddress" - {
-    "return a address" in {
+    "return a address when Connector returns Right" in {
       when(mockSdilConnector.getAddress("123456789")(hc,implicitly)).thenReturn(Future.successful(Right(customerAddressMax)))
 
       val res = service.getAddress("123456789")
 
       whenReady(res) { result =>
-        result mustBe Right(customerAddressMax)
+        result mustBe customerAddressMax
       }
     }
+    "return a exception when Connector returns Left" in {
```

Comment:
For more clarity I think it best to not use Left and Right in the test description, I noticed this on the last PR as well and it is not a massive issue since in reality only devs will look at these so hopefully they should understand Left and Right but describing it what the test expect to see is a better approach.